config/routes.rb

```diff
@@ -113,4 +113,13 @@
     end
   end
 
+  namespace :teachers do
+    get 'schools/:school_id/chart', to: 'charts#show'
```

Comment:
I'm not sure what these routes are for? Do they need to be namespaced?

Reply:
They did, but with fresh eyes, now they don't.